addons/fleet/models/fleet_vehicle_cost.py

```diff
@@ -141,8 +141,8 @@ def default_get(self, default_fields):
         return res
 
     active = fields.Boolean(default=True)
-    vehicle_id = fields.Many2one('fleet.vehicle', 'Vehicle', required=True, help='Vehicle concerned by this log')
-    amount = fields.Float('Cost')
+    vehicle_id = fields.Many2one('fleet.vehicle', 'Vehicle', default=1, required=True, help='Vehicle concerned by this log')
```

Comment:
What is the purpose of this change ? Having a hardcoded id as default value seems very odd.